pyanaconda/ui/lib/software.py

```diff
@@ -19,13 +19,16 @@
 from blivet.arch import is_arm
 from pyanaconda.core.i18n import _
 
+FEATURE_UPSTREAM = "upstream"
 FEATURE_64K = "64k"
-KernelFeatures = namedtuple("KernelFeatures", "page_size_64k")
+
+KernelFeatures = namedtuple("KernelFeatures", "upstream page_size_64k")
```

Comment:
This doesn't look correct. You are defining the `namedtuple` with one string value but then instantiating it with two values. What you probably want is:

```python
KernelFeatures = namedtuple("KernelFeatures", ["upstream", "page_size_64k"])
```

Reply:
Those two statements should be equivalent, see: https://realpython.com/python-namedtuple/#providing-required-arguments-to-namedtuple

Reply:
Ha, interesting. I wasn't aware of that :D

Reply:
I changed the argument to list to fit with the rest of the codebase